src/js/outreach/views/dialog.scss

```diff
@@ -10,10 +10,6 @@
   text-transform: uppercase;
 }
 
-.icon {
-  font-size: 32px;
-}
```

Comment:
Only thing using this was the form submit success icon. But it's no longer being used after that was changed in #1138.

Rest of the icons are using `.dialog__icon > .icon`.